Core/HLE/sceNetInet.cpp

```diff
@@ -50,18 +75,14 @@ static int sceNetInetTerm() {
 }
 
 static int sceNetInetGetErrno() {
-	if (inetLastErrno == 0)
-		inetLastErrno = socket_errno;
```

Comment:
Tuns out that these two lines are critical for MOHH to work, which means we are not properly catching some errno that we really must catch. Fixing this will probably (but not certainly) fix it on Windows too.

Reply:
Btw, `errno` is affected by non-network syscall too, like [strerror for example can change the errno value](https://stackoverflow.com/questions/3870365/are-consecutive-calls-to-errno-to-be-avoided)
But `WSAGetLastError` is different than `GetLastError` function, so it shouldn't be affected by non-winsock functions.

Reply:
Right. But I think what this actually did was rescue us from our bad behavior of resetting the errno on read. If we don't do this, it works just fine.